pkg/noun/manage.c

```diff
@@ -1753,7 +1753,9 @@ _cm_limits(void)
 c3_o
 u3m_backup(c3_o ovw_o)
 {
-  return u3e_backup(ovw_o);
+  c3_c bhk_c[2048];
+  snprintf(bhk_c, 2048, "%s/.urb/bhk", u3P.dir_c);
+  return u3e_backup(bhk_c, ovw_o);
 }
```

Comment:
This function should also be removed, the only call site was for `chop` which no longer uses it.